ncm-download/src/main/pan/components/download/schema.pan

```diff
@@ -9,8 +9,7 @@ type component_download_file = {
       used. Local files can be used as source, such as
       file://localhost/etc/foo.txt or even file:///etc/foo.txt.}
     "href"    : string
-    @{Specify the command (no options allowed) to run
-      whenever the file is updated.
+    @{Specify the command (no options allowed) to run after non-failed download attempt.
```

Comment:
non-failed being different to successful?

Reply:
the code runs the post after a succesful `download` method call. that call can be succesful when nothing was retrieved (eg when timestamp was recent enough). it can also fail when retrieval is ok, but setiing permissions was not. `file is updated` definitely is not correct.
not sure how to phrase it better

Reply:
Makes sense!